bigchaindb/common/transaction.py

```diff
@@ -1150,7 +1201,9 @@ def validate_id(tx_body):
                 tx_body (dict): The Transaction to be transformed.
         """
         # NOTE: Remove reference to avoid side effects
-        tx_body = deepcopy(tx_body)
+        # tx_body = deepcopy(tx_body)
+        # tx_body = rapidjson.loads(rapidjson.dumps(tx_body))
+        tx_body = ujson.loads(ujson.dumps(tx_body))
```

Comment:
:clap: :racehorse: :clap:

I tried your patch with [prof.py](https://gist.github.com/vrde/ce1718c4d646c10e16c4f12776b4fada) and I'm not able to get valid transactions:
```
vrde@node1:~/bigchaindb$ python3 prof.py 1
Create 1 transactions
Start parallel validation
Invalid transaction (InvalidHash): The transaction's id 'd68f7e1cc04788bd5e1492650432fc1759ec793bf5a6d439d4bcb87060944484' isn't equal to the hash of its body, i.e. it's not valid.
  Total time: 0.003963
  Time per transaction: 0.003963
Start serial validation
Invalid transaction (InvalidHash): The transaction's id 'd68f7e1cc04788bd5e1492650432fc1759ec793bf5a6d439d4bcb87060944484' isn't equal to the hash of its body, i.e. it's not valid.
  Total time: 0.044797
  Time per transaction: 0.044797
```

Might be because of this line: `ujson.dumps` should order keys otherwise the hash is incorrect. Also, not using newlines and removing white spaces can be the reason.